ansible_collections/arista/avd/examples/single-dc-l3ls/README.md

```diff
@@ -10,14 +10,19 @@ The example includes and describes all the AVD files and their content used to b
 - Two sets of (virtual) leaf switches, serving endpoints such as servers.
 - Two (virtual) layer2-only switches often used for management connectivity to the servers.
 
-Integration with CloudVision is not included in this example to keep everything as simple as possible. In this case, the Ansible host will communicate directly with the switches using eAPI.
+Ansible playbooks are included to show the following:
+
+- Building the intended configuration and documentation
+- Deploying the configuration directly to the switches using eAPI
+- Deploying the configuration via CloudVision to the switches, thereby including a full change-based workflow with rollback capability etc.
 
 ## Installation
 
 Requirements to use this example:
 
 - Follow the installation guide for AVD found [here](../../docs/installation/collection-installation.md).
-- Run the following playbook to copy the examples to your current working directory, for example `ansible-avd-examples`:
+- If you intend to use the integration with CloudVision, follow the separate installation guide for the CVP collection [here](https://cvp.avd.sh/en/latest/docs/installation/collection-installation/).
```

Comment:
Not needed, since the CVP collection is already part of the dependencies for AVD.

Reply:
Fixed